.github/workflows/build.yml

```diff
@@ -415,104 +246,23 @@ jobs:
           name: terraform_${{env.version}}_${{ env.os }}_${{ env.arch }}.zip
           path: .
       - name: Extract packages
+        if: ${{ (matrix.goos == 'linux') || (inputs.goos == 'darwin') }}
         run: |
           unzip "./terraform-e2etest_${{ env.os }}_${{ env.arch }}.zip"
```

Comment:
@terraform can you confirm that these are the correct e2e tests to run? In the `terraform-releases` Circle CI pipeline, the e2e tests are ran by downloading and executing a `terraform-exec` binary. Do we need to include that here?

Reply:
It would be good to include both if possible. We (as in terraform-core) can also come in later and add the terraform-exec e2e tests once the pipeline is up and running.